package.json

```diff
@@ -48,6 +50,7 @@
     "jsdom-testing-mocks": "^1.13.0",
     "msw": "^2.3.4",
     "msw-storybook-addon": "^2.0.3",
+    "playwright": "^1.46.0",
```

Comment:
That's interesting, playwright is already in the deps of `@storybook/test-runner`, but I understand we need to have it in our deps as well in order for `pnpm exec playwright install` to work, right? https://github.com/pnpm/pnpm/issues/899

Reply:
@ravicious I'm not sure if this is exactly the issue here, but yes, simply installing the test runner didn't allow me to run the tests, since Playwright complained about missing binaries.